<launch>

  <node pkg="project3_phase2" type="control_node.py" name="controller" output="screen">
    <param name="linear_k_p" value="0.5"/>
    <param name="linear_k_d" value="20"/>
    <param name="linear_k_i" value="0.001"/>
    <param name="angular_k_p" value="4"/>
    <param name="angular_k_d" value="40"/>
    <param name="angular_k_i" value="0"/>
    <param name="dt" value="0.001"/>
    <param name="epsilon" value="0.005"/>
    <param name="path_type" value="3"/>
  </node>

  <node pkg="project3_phase2" type="monitor.py" name="monitor" ></node>

  <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_empty_world.launch">
    <arg name="x_pos" value="0.0"/>
    <arg name="y_pos" value="0.0"/>
    <arg name="z_pos" value="0.0"/>

  </include>  

  <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_gazebo_rviz.launch">
  </include>
</launch>
